### PR TITLE
Restore string to cacheLocation type

### PR DESCRIPTION
**Pull request by jasonnutter**
Unfortunately, using the value of a value in an enum is not good enough for typescript, so we need to restore `string` as a possible type. The `BrowserCacheManager` will [handle a string literal value correctly](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/src/cache/BrowserCacheManager.ts#L52-L66) at runtime.

### CODE REVIEW

File: change/@azure-msal-browser-2020-10-29-16-03-05-msal-browser-cache-location-string.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Restore string to cacheLocation type",
```
Comment (jasonnutter) [inline change/@azure-msal-browser-2020-10-29-16-03-05-msal-browser-cache-location-string.json:3]:
```suggestion
  "comment": "Restore string to cacheLocation type (#2523)",
```

File: change/@azure-msal-browser-2020-10-29-16-03-05-msal-browser-cache-location-string.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "packageName": "@azure/msal-browser",
+  "email": "janutter@microsoft.com",
+  "dependentChangeType": "patch",
```
Comment (jasonnutter) [inline change/@azure-msal-browser-2020-10-29-16-03-05-msal-browser-cache-location-string.json:6]:
```suggestion
  "dependentChangeType": "none",
```

**Review by pkanher617 — APPROVED** [review]
LGTM

**Review by jo-arroyo — APPROVED** [review]

**Review by sameerag — APPROVED** [review]
So it is a String `enum` value compared to user input (string) runtime?

**Pull request merged**